packages/dx-react-scheduler/docs/reference/week-view.md

```diff
@@ -56,6 +56,7 @@ Field | Type | Description
 ------|------|------------
 startDate | Date | Specifies the cell start time.
 endDate | Date | Specifies the cell end time.
+today | boolean | Specifies the cell's date is today.
```

Comment:
Is it necessary for WeekView?

Reply:
This is the `DayView.CellData` interface. The `cellsData` function always returns this interface.